modules/adriverBidAdapter.js

```diff
@@ -99,22 +97,17 @@ export const spec = {
       });
     });
 
-    let userid = validBidRequests[0].userId;
     let adrcidCookie = storage.getDataFromLocalStorage('adrcid') || validBidRequests[0].userId.adrcid;
```

Comment:
During my tests, I saw the fallback throw an error during the first auction, because the userId is not defined.

Please just update to something like : `|| validBidRequests[0].userId?.adrcid;`